docs/contributing/ui-testing.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Ui Testing
+---
+## Prerequisites
+Clone sandstone from GitHub, install dependencies and connect the modules using Lerna:
+
+```shell
+# clone the repo!
+git clone git@github.com:enactjs/sandstone.git
+# move in
+cd sandstone
+# we're using git flow so develop is our working branch
+git checkout develop
+# install lerna
+npm install
+# link dependencies
+enact link
+```
+
+## Creating tests
+Within the UI Library, create an app for testing in `./tests/ui/apps` and create a corresponding test in `./tests/ui/specs`.
+
+	+ src
+		+ test
+			+ ui
+				+ apps
+					+testComponent
+						testComponent-View.js		<-- create app here
+				+ specs
+					+ testComponent
+						testComponent-specs.js		<-- create spec file here
+ In UI test, create apps to test view and specs. Please refer sample code.
+* Button.js
+
+```JS
+import Button from '../../../../Button';
+import ThemeDecorator from '../../../../ThemeDecorator';
+
+const app = (props) => <div {...props}>
+	<div>
+		<Button
+			id="button"
+		>
+			Default Button
+		</Button>
+	</div>
+</div>;
+
+export default ThemeDecorator(app);
+```
+
+* Button-specs.js
+
+```JS
+const Page = require('@enact/ui-test-utils/utils');
+
+describe('Button', function () {
+
+	beforeEach(async function () {
+		await Page.open();
+	});
+
+	describe('5-way', function () {
+		it('should focus disabled button on 5-way right', async function () {
+			await Page.spotlightDown();
+			await Page.spotlightSelect();
+		});
+	});
+});
+```
+
+The Page component from `@enact/ui-test-utils/utils/` Page contains useful methods for loading tests.
```

Comment:
```suggestion
The Page component from `@enact/ui-test-utils/utils/` Page contains useful methods for writing tests.
```

I don't know what is mean of loading.

Reply:
I removed this part and added link for ui-test-utils.